packages/fx-core/src/common/local/constants.ts

```diff
@@ -98,6 +98,7 @@ export const TaskCommand = Object.freeze({
   setUpBot: "debug-set-up-bot",
   setUpSSO: "debug-set-up-sso",
   prepareManifest: "debug-prepare-manifest",
+  launchTeamsWebClient: "debug-launch-teams-client",
```

Comment:
Will this be used to launch outlook and m365 app later?

Reply:
And, it seems this task is not for local only, and can also be used for remote preview. So how about removing the `debug-` prefix?

Reply:
- Removing `debug-` prefix make sense to me.
- We're still considering whether to support m365 app later, and we can first add support for Teams. If m365 support is needed we can add PRs later for launching other hubs (and the command name will change to something like `launch-m365-hub`)

Reply:
Rename a task command is a bit heavy... So I'd suggest the task command be something like `launch-web-client`, and make the task label in tasks.json more specific, e.g., `Launch Teams Web Client`.